<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddCoupureDialog</class>
 <widget class="QDialog" name="AddCoupureDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Début du coupure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="startDate">
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="startTime">
       <property name="displayFormat">
        <string>hh:mm</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
       <property name="time">
        <time>
         <hour>12</hour>
         <minute>0</minute>
         <second>0</second>
        </time>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Fin du coupure</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDateEdit" name="endDate">
       <property name="displayFormat">
        <string>d/M/yyyy</string>
       </property>
       <property name="calendarPopup">
        <bool>true</bool>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTimeEdit" name="endTime">
       <property name="displayFormat">
        <string>hh:mm</string>
       </property>
       <property name="timeSpec">
        <enum>Qt::LocalTime</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Consommation du groupe electrogène</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="conso_spin">
       <property name="maximum">
        <number>99999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Prix unitaire du carburant</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="prix_spin">
       <property name="maximum">
        <number>99999999</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="saveBtn">
     <property name="text">
      <string>Enregistrer</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
